commands/push.js

```diff
@@ -33,6 +34,12 @@ module.exports = function (topic) {
         name: 'arg',
         hasValue: true,
         description: 'set build-time variables'
+      },
+      {
+        name: 'path',
```

Comment:
`--path` is a bit obscure IMHO. I'd name this `--context`, or maybe even `--context-path`.

Reply:
Agreed, will change to `--context-path`